mehmetsaidozsoy.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "Mehmet Said ÖZSOY",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": " Motorcycle lover",
+  "avatar": "https://avatars.githubusercontent.com/u/6255955?s=40&v=4",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/OzsoySaid",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/mehmetsaidozsoy_/",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/mehmetsaidozsoy",
+      "icon": "FaInstagram"
```

```suggestion
      "icon": "FaGithub"
```
You could use the GitHub icon for your GitHub profile